chia/consensus/blockchain.py

```diff
@@ -926,27 +926,18 @@ async def get_header_block_by_height(
             return None
         return header_dict[header_hash]
 
-    async def get_block_records_at(self, heights: list[uint32], batch_size: int = 900) -> list[BlockRecord]:
+    async def get_block_records_at(self, heights: list[uint32]) -> list[BlockRecord]:
```

Comment:
We don't actually want a `batch_size` parameter here: it should be an implementation detail.